DXMainClient/PreStartup.cs

```diff
@@ -73,6 +74,53 @@ public static void Initialize(StartupParams parameters)
 
             UserINISettings.Initialize(ClientConfiguration.Instance.SettingsIniName);
 
+            // Try to load translations
+            try
+            {
+                var translation = TranslationTable.LoadFromIniFile(ClientConfiguration.Instance.TranslationIniName);
+                LanguageInstance.TranslationTable = translation;
+                Logger.Log("Load translation: " + translation.LanguageName);
+            }
+            catch (Exception ex)
+            {
+                Logger.Log("Failed to load the translation file. " + ex.Message);
+            }
+
+            try
+            {
+                if (ClientConfiguration.Instance.GenerateTranslationStub)
+                {
+                    // Not use using statement as variable "fs" will be destroyed after the lambda function be destroyed
+                    var fs = File.Open("Client/Translation.stub.ini", FileMode.Create);
+                    var sw = new StreamWriter(fs, new System.Text.UTF8Encoding(false)) { AutoFlush = true };
```

Comment:
This is because when a label is missing, the ini file needs to be re-written. Use an INI class or TranslationTable.SaveIni() causes the file to be totally re-written every time a new label is missing, instead of just append the value to the end at the text stream.